src/components/Profile/styles.scss

```diff
@@ -0,0 +1,22 @@
+.flex-row {
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: left;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    margin: 0 auto;
+}
+
+.flex-item {
```

Comment:
CSS classes should describe what some component is: a class could be e.g. title, title icon, description, box-header etc